src/types/onyx/Fund.ts

```diff
@@ -0,0 +1,17 @@
+import {ValueOf} from 'type-fest';
+import CONST from '../../CONST';
+
+type Fund = {
```

Comment:
Not sure how in depth TS definitions should go, but if this is eventually a list of slightly different payment card types, should I specify these to be "Expensify Card" specific and have Fund be a union of slightly different card object shapes? 


cc @hayata-suenaga on TS question

Reply:
Kind of a pre-optimization though, we could also determine when we get to it

Reply:
This conversation should move under "Card.ts", I swapped the types

Reply:
Once again here, these are not Expensify Cards 😄

Reply:
this thread was started before [this](https://github.com/Expensify/App/pull/25939/commits/a290a6f8ffb974944eefdb3260b84d2720c75202) commit, sorry for the confusion!